### PR TITLE
Remove unused code and usage stats from welcome flow

### PR DESCRIPTION
**Pull request by sergiou87**
## Description

This PR makes a few changes in the welcome flow.

### Screenshots

image

## Release notes

Notes: no-notes

### CODE REVIEW

File: app/src/ui/welcome/configure-git.tsx [hunk 1]
```diff
@@ -7,6 +7,7 @@ import { Button } from '../lib/button'
 interface IConfigureGitProps {
   readonly accounts: ReadonlyArray<Account>
   readonly advance: (step: WelcomeStep) => void
+  readonly done: () => void
```
Comment (tidy-dev) [inline app/src/ui/welcome/configure-git.tsx:10]: I realize this was the already an establish verbage.. but `done` feels like a weird word for an action and took me a second to read it. Feel free to ignore this (probably just me).

**Review by tidy-dev — APPROVED** [review]
I had one code nit.. but not anything to prevent this from going in. From a code perspective, this looks great. :)

**Review by billygriffin — APPROVED** [review]
Looks good to me!

File: app/src/ui/welcome/configure-git.tsx (hunk 1 above)
Comment (sergiou87) [reply app/src/ui/welcome/configure-git.tsx:10]: Yep, I agree 😆  But as you said I basically copied/moved this from the usage stats screen 😅  So I'm not gonna change it now if that's ok!

Comment (a420gmoon) [reply app/src/ui/welcome/configure-git.tsx:10]: As you know, I going to tell you the right thing, I am not going to wrong Turn!

**Review by 641210 — COMMENTED** [review]
****

**Pull request merged**